api/src/main/scala/pl/touk/nussknacker/engine/api/exception/NuExceptionInfo.scala

```diff
@@ -1,5 +1,16 @@
 package pl.touk.nussknacker.engine.api.exception
 
 import pl.touk.nussknacker.engine.api.Context
+import pl.touk.nussknacker.engine.api.component.ComponentType.ComponentType
 
-case class NuExceptionInfo[T <: Throwable](nodeId: Option[String], throwable: T, context: Context) extends Serializable
\ No newline at end of file
+case class NuExceptionInfo[T <: Throwable](nodeComponentId: Option[NodeComponentInfo], throwable: T, context: Context) extends Serializable
```

Comment:
should be `nodeComponentInfo` after class rename

Reply:
Fixed